Assets/MixedRealityToolkit/UX/Resources/Dialog.prefab

```diff
@@ -0,0 +1,2459 @@
+%YAML 1.1
```

Comment:
Please move anything in resources folder to appropriate place. Resources folder was removed.